test/tests/providers/global/loki.all.rules.test.yml

```diff
@@ -0,0 +1,99 @@
+---
+rule_files:
+  - loki.all.rules.yml
+
+tests:
+  - interval: 1m
+    input_series:
+      - series: 'cortex_ring_members{app="loki-compactor", cluster_id="zj88t", cluster_type="workload_cluster", container="compactor", customer="giantswarm", installation="gorilla", instance="10.7.116.221:3100", job="zj88t-prometheus/workload-zj88t/0", name="compactor", namespace="loki", node="ip-10-6-2-178.eu-central-1.compute.internal", organization="giantswarm-production", pod="loki-compactor-676b8c897b-rq298", provider="aws", service_priority="highest", state="Unhealthy"}'
+        values: "0+0x20 1+0x160"  # 1 unhealthy value after 20 minutes
+      - series: 'loki_panic_total{app="loki-compactor", cluster_id="zj88t", cluster_type="workload_cluster", container="compactor", customer="giantswarm", installation="gorilla", instance="10.7.116.221:3100", job="zj88t-prometheus/workload-zj88t/0", namespace="loki", node="ip-10-6-2-178.eu-central-1.compute.internal", organization="giantswarm-production", pod="loki-compactor-676b8c897b-rq298", provider="aws", service_priority="highest"}'
+        values: "0+0x20 1+0x160"  # 1 panic after 20 minutes
+      - series: 'loki_request_duration_seconds_count{app="loki-distributor", cluster_id="zj88t", cluster_type="workload_cluster", container="distributor", customer="giantswarm", installation="gorilla", instance="10.7.75.90:3100", job="zj88t-prometheus/workload-zj88t/0", method="POST", namespace="loki", node="ip-10-6-2-141.eu-central-1.compute.internal", organization="giantswarm-production", pod="loki-distributor-74b78f5559-tz6zs", provider="aws", route="loki_api_v1_push", service_priority="highest", status_code="204", ws="false"}'
+        values: "0+60x180"  # 1 request per second OK for 3 hours
```

Comment:
IIUC `0+60x180` means we add 60 to each sample, giving us something like

```
0 60 120 180 240 ...
```

I read this as we have 60 request per second for 3 hours, am I wrong ?

Reply:
We defined `interval: 1m`, so it's one sample per minute. So 60 requests per minute.